src/locales/en.json

```diff
@@ -20,5 +20,15 @@
     "unable.to.parse.build.gradle": "Unable to parse build.gradle: {0}.  The error is {1}",
     "error.parsing.pom": "Unable to parse pom file, the error is {0}",
     "refreshing.liberty.dashboard": "Refreshing Liberty Dashboard projects...",
-    "no.liberty.project.found.in.empty.workspace": "No Liberty project found in empty workspace"
+    "no.liberty.project.found.in.empty.workspace": "No Liberty project found in empty workspace",
+    "add.project.manually.no.projects.available.to.add": "No workspace folders available to be added to Liberty Dashboard",
+    "add.project.manually.message.0": "The selected project has been successfully added to Liberty Dashboard",
+    "add.project.manually.message.1": "The selected project already exists in Liberty Dashboard",
+    "add.project.manually.message.2": "The selected project is not a Maven or Gradle project",
+    "confirmation.button.label.yes": "Yes",
+    "confirmation.button.label.no": "No",
+    "remove.custom.project.confirmation": "Are you sure you want to remove the project in {0} from Liberty Dashboard?",
```

Comment:
should say "from the Liberty Dashboard?"

Reply:
fixed